src/openforms/config/service.py

```diff
@@ -0,0 +1,39 @@
+from dataclasses import dataclass
+
+from django.utils.translation import gettext as _
+
+from openforms.contrib.brk.checks import BRKValidatorCheck
+from openforms.forms.models.form import Form
+from openforms.plugins.exceptions import InvalidPluginConfiguration
+
+
+@dataclass
+class BrokenConfiguration:
+    config_name: str
```

Comment:
You can use `StrOrPromise` from `openforms.typing`

```suggestion
    config_name: StrOrPromise
```

`gettext_lazy` returns a `Promise` object, so our `StrOrPromise` is a convenient type alias that accepts both `str` and `Promise` objects.

You'll also have to update the `gettext` import to use `gettext_lazy`